### PR TITLE
Detail page animations #14

### LINKED ISSUE #14: Detail page animations

### PR DESCRIPTION
**Pull request by DphnZwp**

### CODE REVIEW

File: src/routes/stories/[id]/+page.svelte [hunk 1]
```diff
@@ -157,7 +213,7 @@
 		border-radius: 0.7rem;
 		padding: 1rem 2rem;
 		filter: contrast(110%) brightness(100%);
-		background: linear-gradient(40deg, rgba(195, 165, 128, 1), rgba(0, 0, 0, 0)),
+		background: linear-gradient(40deg, #b38c5d, #00000000),
```
Comment (JustinLung) [inline src/routes/stories/[id]/+page.svelte]: Use css variables here for the colors

File: src/routes/stories/[id]/+page.svelte [hunk 2]
```diff
@@ -10,87 +10,137 @@
...
+				trigger: '.story__content *:nth-child(12)',
+				toggleActions:"restart none restart restart"
+			},
+			delay: .2,
+			x: 100,
+			ease: "back.out(1)",
+			duration: 1.25,
+		});
 	});
-	// onMount(() => {
```
Comment (JustinLung) [inline src/routes/stories/[id]/+page.svelte:27]: Remove unused code

**Review by JustinLung — CHANGES_REQUESTED** [review]

File: src/routes/stories/[id]/+page.svelte (hunk 2 above)
Comment (DphnZwp) [reply src/routes/stories/[id]/+page.svelte:27]: There is no unused code for the onMount. Ik see no problems on line 27.

Comment (JustinLung) [reply src/routes/stories/[id]/+page.svelte:27]: Oh ja my bad, merge maar

**Pull request merged**